src/main/flight/mixer_tricopter.c

```diff
@@ -95,7 +95,7 @@
 #define TRI_YAW_FORCE_PRECISION     (1000)
 
 #define IsDelayElapsed_us(timestamp_us, delay_us) ((uint32_t)(micros() - timestamp_us) >= delay_us)
-#define IsDelayElapsed_ms(timestamp_ms, delay_ms) ((uint32_t)(millis() - timestamp_ms) >= delay_ms)
+#define IsDelayElapsed_ms(timestamp_ms, delay_ms) ((uint32_t)(now - timestamp_ms) >= delay_ms)
```

Comment:
This macro can now be used only in a function that defines 'now' correctly. I prefer the millis() better